releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,29 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 6.0.0 - March 28, 2023
+### Breaking Changes:
```

Comment:
Looks like we dont have any bug fixes to call out this time. Since they are not all breaking changes, we can create a new section for the non-breaking changes (last two points in your list) and call it "Enhancements".